R/pkg/.lintr

```diff
@@ -1,2 +1,2 @@
-linters: with_defaults(line_length_linter(100), multiple_dots_linter = NULL, object_name_linter = NULL, camel_case_linter = NULL, open_curly_linter(allow_single_line = TRUE), closed_curly_linter(allow_single_line = TRUE))
+linters: with_defaults(line_length_linter(100), multiple_dots_linter = NULL, object_name_linter = NULL, camel_case_linter = NULL, open_curly_linter(allow_single_line = TRUE), closed_curly_linter(allow_single_line = TRUE), object_usage_linter = NULL)
```

Comment:
`object_usage_linter` has a limitation which cannot detect `shell` function used in `Windows` environment. Also, `nolint` doesn't work for this rule. This is a known limitation of lint-r itself.
- https://github.com/jimhester/lintr/issues/27